src/app/screens/sendRareSat/index.tsx

```diff
@@ -19,85 +14,92 @@ import {
   validateBtcAddress,
 } from '@secretkeylabs/xverse-core';
 import { useMutation } from '@tanstack/react-query';
-import { StyledHeading, StyledP } from '@ui-library/common.styled';
-import { isLedgerAccount } from '@utils/helper';
-import { getBundleId, getBundleSubText } from '@utils/rareSats';
+import Callout from '@ui-library/callout';
+import { StyledHeading } from '@ui-library/common.styled';
+import InputFeedback, { InputFeedbackProps, isDangerFeedback } from '@ui-library/inputFeedback';
 import BigNumber from 'bignumber.js';
-import { useEffect, useMemo, useState } from 'react';
+import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useLocation, useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
+import SendLayout from '../../layouts/sendLayout';
 
-const ScrollContainer = styled.div`
+const Container = styled.div`
   display: flex;
-  flex: 1;
   flex-direction: column;
-  overflow-y: auto;
-  &::-webkit-scrollbar {
-    display: none;
-  }
-  width: 360px;
-  margin: auto;
+  justify-content: space-between;
+  flex-grow: 1;
+  padding: 0 ${(props) => props.theme.space.m};
```

Comment:
not a blocker: padding too large
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/0e4ee9ba-6a80-42ae-ad1c-48d742ccaafe)

Reply:
send nft: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/b0cfe295-8155-4e77-bc57-42d0b9e07ac2)